src/app_service/service/community_tokens/service.nim

```diff
@@ -77,9 +77,9 @@ QtObject:
       let contractGasUnits = "3702411"
 
       let txData = ens_utils.buildTransaction(parseAddress(addressFrom), 0.u256, contractGasUnits,
-      if suggestedFees.eip1559Enabled: "" else: $suggestedFees.gasPrice, suggestedFees.eip1559Enabled,
-      if suggestedFees.eip1559Enabled: $suggestedFees.maxPriorityFeePerGas else: "",
-      if suggestedFees.eip1559Enabled: $suggestedFees.maxFeePerGasM else: "")
+        if suggestedFees.eip1559Enabled: "" else: $suggestedFees.gasPrice, suggestedFees.eip1559Enabled,
+        if suggestedFees.eip1559Enabled: $suggestedFees.maxPriorityFeePerGas else: "",
+        if suggestedFees.eip1559Enabled: $suggestedFees.maxFeePerGasM else: "")
```

Comment:
The indentation looks a bit off here

Reply:
Jo asked about it in the prev pr to indicate that these lines belongs to txData.

Reply:
ohh I see ti now